controller/src/beerocks/master/db/db.h

```diff
@@ -394,6 +394,42 @@ class db {
     bool set_hostap_is_acs_enabled(std::string mac, bool enable);
     bool get_hostap_is_acs_enabled(std::string mac);
 
+    //
+    // Channel Scan
+    //
+    bool set_channel_scan_is_enabled(const std::string &mac, const bool enable);
```

Comment:
I prefer `bool channel_scan_enable(const std::string &mac, bool enable);`
Same for all other APIs.
(nitpick)

Reply:
most other function in the db follow the set_X get_X signature style